index.js

```diff
@@ -1157,6 +1157,10 @@ class Encore {
         return this;
     }
 
+    isRuntimeConfigured() {
```

Comment:
Question here: should it be named `isRuntimeConfigured` or `isRuntimeEnvironmentConfigured` ? Other methods are taking about `RuntimeEnvironment`.